kafka_consumer/datadog_checks/kafka_consumer/kafka_consumer.py

```diff
@@ -238,6 +248,42 @@ def _get_group_coordinator(self, client, group):
 
         return coord_id
 
+    def _get_kafka_consumer_groups(self, client, kafka_conn_str):
+        # Until kafka-python implements the adminClient that
+        # can retrieve list of consumer groups we will be using this
+        # method adapted from https://github.com/amoma-com/zabbix-kafka-consumers-monitor
+        group_request = admin.ListGroupsRequest_v0(
+            timeout=self._kafka_timeout
+        )
+        consumer_group = {}
+        kafka_broker_ids = [broker.nodeId for broker in client.cluster.brokers()]
+        for b_id in kafka_broker_ids:
+            tries = 0
+            max_tries = 5
+            data_from_node = False
+
+            while not data_from_node and tries <= max_tries:
+                future = client.send(b_id, group_request)
+                client.poll(timeout_ms=self._kafka_timeout, future=future)
+                if future.value is not None:
+                    results = future.value.groups
+
+                    for r in results:
+                        topics = {}
+                        # Get topics for a consumer group
+                        consumer = kafka.KafkaConsumer(group_id=r[0], bootstrap_servers=[kafka_conn_str])
```

Comment:
@ChristineTChen does this trigger a consumer group rebalance? I know it's the only way to figure out the mapping between consumer groups / topics (at least until KIP-88 landed in Kafka 0.10.2), but I've always avoided implementing this because I thought it triggered a rebalance, which can be quite expensive because it temporarily brings the whole consumer group to a stop...

Reply:
@jeffwidman Thanks for taking the time to look over this PR! I wasn't aware of the rebalancing issue caused from adding new consumers. Looking into the test kafka instance, a rebalance does occur. 

I agree with your suggestion for supporting only versions >=0.10.2, I'll hold off on this PR until your feature is merged upstream.